pitch.md

```diff
@@ -1,60 +1,142 @@
-# Project Title (replace me)
+# To-Do List Organizer
 
 ## Describe your project in five words
+Chronological to-do lists from categories
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
-them?_
+them?_  
+The audience would be people who use to-do lists to organize their plans.
+Currently, either people organize a to-do list chronologically, or they do it by some bigger, more general
+categories. In ordering lists chronologically, it's easy to miss items as you have to think back and forth
+between all the categories. When putting to-do lists together by categories, on the other hand, it's hard to
+see it as a cohesive to-do list instead of multiple to-do lists that you need to juggle. In other words, it's
+easier to write but harder to use.
 
 
 ## Why a language?
-_Why is a **DSL** appropriate for your user(s)? How does it address the need?_
+_Why is a **DSL** appropriate for your user(s)? How does it address the need?_  
+It is helpful because the language can allow for a lot of flexibility in what the user wants to be able to
+write. Additionally, categorized to-do lists are already like a language. You have a large category with
```

Comment:
It definitely felt language like when you described how users can have a lot of flexibility over how they want to specify the due date. Allowing them to say Monday or Monday at 5 or 10/24 seems powerful